### PR TITLE
Update TorGuard to 4.8.2

### PR DESCRIPTION
**Pull request by auanasgheps**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/denelon/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/34628)

### CODE REVIEW

**Comment by auanasgheps** [comment]
Dang! Will double-check and make appropriate changes.

File: manifests/t/TorGuard/TorGuard/4.8.2/TorGuard.TorGuard.yaml [hunk 1]
```diff
@@ -2,7 +2,6 @@
 # yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.0.0.schema.json
 
 PackageIdentifier: TorGuard.TorGuard
-PackageVersion: 4.8.1-build.106+g8894abc
-DefaultLocale: en-US
+PackageVersion: 4.8.2
```
Comment (Trenly) [inline manifests/t/TorGuard/TorGuard/4.8.2/TorGuard.TorGuard.yaml:5]:
```suggestion
PackageVersion: 4.8.2
DefaultLocale: en-US
```

File: manifests/t/TorGuard/TorGuard/4.8.2/TorGuard.TorGuard.installer.yaml [hunk 2]
```diff
@@ -2,7 +2,7 @@
 # yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
 
 PackageIdentifier: TorGuard.TorGuard
-PackageVersion: 4.8.1-build.106+g8894abc
+PackageVersion: 4.8.2
```
Comment (Trenly) [inline manifests/t/TorGuard/TorGuard/4.8.2/TorGuard.TorGuard.installer.yaml:5]:
```suggestion
PackageVersion: 4.8.2-build.111+gb4c9c59
```

File: manifests/t/TorGuard/TorGuard/4.8.2/TorGuard.TorGuard.locale.en-US.yaml [hunk 3]
```diff
@@ -2,7 +2,7 @@
 # yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
 
 PackageIdentifier: TorGuard.TorGuard
-PackageVersion: 4.8.1-build.106+g8894abc
+PackageVersion: 4.8.2
```
Comment (Trenly) [inline manifests/t/TorGuard/TorGuard/4.8.2/TorGuard.TorGuard.locale.en-US.yaml:5]:
```suggestion
PackageVersion: 4.8.2-build.111+gb4c9c59
```

File: manifests/t/TorGuard/TorGuard/4.8.2/TorGuard.TorGuard.yaml (hunk 1 above)
Comment (Trenly) [inline manifests/t/TorGuard/TorGuard/4.8.2/TorGuard.TorGuard.yaml:5]:
```suggestion
PackageVersion: 4.8.2-build.111+gb4c9c59
DefaultLocale: en-US
```

File: manifests/t/TorGuard/TorGuard/4.8.2/TorGuard.TorGuard.locale.en-US.yaml [hunk 4]
```diff
@@ -2,7 +2,7 @@
...
 PackageLocale: en-US
 Publisher: VPNetwork LLC
```
Comment (Trenly) [inline manifests/t/TorGuard/TorGuard/4.8.2/TorGuard.TorGuard.locale.en-US.yaml:7]:
```suggestion
Publisher: '"VPNetwork LLC"'
```

**Review by Trenly — CHANGES_REQUESTED** [review]
The version needs to match the one in control panel

**Comment by auanasgheps** [comment]
> The version needs to match the one in control panel

Thanks, I wasn't aware of this.

**Comment by OfficialEsco** [comment]
```
Manifest: The manifest file path must match PackageIdentifier and PackageVersion properties from manifest. 
actual 'manifests\t\TorGuard\TorGuard\4.8.2', 
expected 'manifests\t\TorGuard\TorGuard\4.8.2-build.111+gb4c9c59
```

**Review by ItzLevvie — APPROVED** [review]

**Pull request merged**